src/Core/Addon/Theme/Theme.php

```diff
@@ -42,6 +42,10 @@ public function __construct(array $attributes)
             $parentAttributes['preview'] = 'themes/' . $attributes['parent'] . '/preview.png';
             $parentAttributes['parent_directory'] = rtrim($attributes['directory'], '/') . '/';
             $attributes = array_merge($parentAttributes, $attributes);
+
+            if (isset($attributes['assets']['use_parent_assets']) && $attributes['assets']['use_parent_assets'] && isset($parentAttributes['assets']) {
```

Comment:
```suggestion
            if (!empty($attributes['assets']['use_parent_assets']) && isset($parentAttributes['assets']) {
```

Reply:
Missing `)` not sure this code was tested correctly :thinking:

Reply:
my bad :(
true that is, I will be more careful next time